servicetalk-grpc-netty/build.gradle

```diff
@@ -32,7 +32,7 @@ dependencies {
   api project(":servicetalk-grpc-utils")
   api project(":servicetalk-http-netty")
 
-  implementation project(":servicetalk-http-utils")
+  compileOnly project(":servicetalk-http-utils")
```

Comment:
only timeout filter from this project is referenced and just from javadocs...

Reply:
Consider either adding a comment here so we don't forget or changing javadoc from using `@link` to `@code` with full class name, then you can kill this line.

Reply:
I'll add a comment here. `{@link}` ensures code doesn't become stale and makes javadocs easier to navigate.